.chronus/changes/uri-templates-2024-6-25-9-3-39.md

```diff
@@ -0,0 +1,13 @@
+---
+changeKind: deprecation
+packages:
+  - "@typespec/http"
+---
+
+Deprecated `@query({format: })` option. Use `@query(#{explode: true})` instead of `form` or `multi` format. Previously `csv`/`simple` is the default now.
```

Comment:
Should also include the deprecation in the `format` property returned by the accessor to emitters